msal/src/Microsoft.Identity.Client/ClientApplicationBase.cs

```diff
@@ -82,30 +97,44 @@ public abstract partial class ClientApplicationBase
         }
 
         /// <summary>
-        /// Identifier of the component consuming MSAL and it is intended for libraries/SDKs that consume MSAL. This will allow for disambiguation between MSAL usage by the app vs MSAL usage by component libraries.
+        /// Identifier of the component (libraries/SDK) consuming MSAL.NET. 
+        /// This will allow for disambiguation between MSAL usage by the app vs MSAL usage by component libraries.
         /// </summary>
         public string Component { get; set; }
 
         /// <Summary>
-        /// Authority provided by the developer or default authority used by the library.
+        /// Gets the URL of the authority, or Security Token service (STS) from which MSAL.NET will acquire security tokens
+        /// The return value of this propety is either the value provided by the developer in the constructor of the application, or otherwise 
+        /// the value of the <see cref="DefaultAuthority"/> static member (that is <c>https://login.microsoftonline.com/common/</c>)
         /// </Summary>
         public string Authority { get; }
```

Comment:
>Authority [](start = 22, length = 9)

aka.ms... where we talk more about Authorities?

Reply:
Done in the summary of the class